### PR TITLE
[14.0][IMP] mrp_subcontracting_partner_management

### PR DESCRIPTION
**Pull request by geomer198**
Dependencies and module flow is updated.

### CODE REVIEW

**Review by francesco-ooops — APPROVED** [review]
LGTM

@geomer198 please squash

**Review by GabbasovDinar — APPROVED** [review]

**Comment by francesco-ooops** [comment]
@pedrobaeza what do you think?

File: mrp_subcontracting_partner_management/__manifest__.py [hunk 1]
```diff
@@ -6,7 +6,8 @@
     "license": "LGPL-3",
     "category": "Inventory",
     "website": "https://github.com/OCA/manufacture",
-    "depends": ["purchase_stock", "mrp_subcontracting", "sale_stock"],
+    "depends": ["purchase_stock", "mrp_subcontracting_purchase", "sale_stock"],
```
Comment (pedrobaeza) [inline mrp_subcontracting_partner_management/__manifest__.py]: You shouldn't change dependencies in a stable module. Do instead an extension module.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by geomer198** [comment]
@pedrobaeza Tests in the current module is broken in bridge module. I propose a solution: Can I add  in settings a parameter for the activate `mrp_subcontracting_purchase` module and change a tests with depends on installing this module.

**Comment by pedrobaeza** [comment]
You can do both modules in this PR.

**Pull request closed without merging**